src/mudblazor/MudBlazor.Template/Components/Account/Pages/_Imports.razor

```diff
@@ -1,3 +1,3 @@
 ﻿@using MudBlazor
 @using MudBlazor.Template.Components.Account.Shared
-@layout AccountLayout
```

Comment:
Interestingly, the entire `AccountLayout.razor` was deleted in the latest templates, and `MainLayout.razor` is used instead.